web/packages/teleport/src/Account/Account.tsx

```diff
@@ -163,13 +168,197 @@ export function Account({
     onDeviceAdded();
   }
 
+  let ctx = useContext(UserContext);
+
+  let layouts = {
```

Comment:
This list doesn't change, let's keep it outside the React component. Now the object is recreated on every rerender (and the code is harder to read).